protocol/x/clob/keeper/orders.go

```diff
@@ -130,18 +131,36 @@ func (k Keeper) PlaceShortTermOrder(
 func (k Keeper) CancelStatefulOrder(
 	ctx sdk.Context,
 	msg *types.MsgCancelOrder,
-) error {
+) (err error) {
+	defer func() {
+		if err != nil {
+			callback := metrics.DeliverTx
+			if ctx.IsCheckTx() {
+				callback = metrics.CheckTx
+			}
+			if ctx.IsReCheckTx() {
+				callback = metrics.ReCheckTx
+			}
+			telemetry.IncrCounterWithLabels(
+				[]string{types.ModuleName, metrics.CancelStatefulOrder, metrics.Error, metrics.Count},
+				1,
+				[]gometrics.Label{
+					metrics.GetLabelForStringValue(metrics.Callback, callback),
+				},
+			)
```

Comment:
Where do you emit a success metric? Ideally we have enough metrics to get a % success rate for stateful order placements + cancellations

Reply:
will add this in